javascript/organizing_your_javascript_code/webpack.md

```diff
@@ -408,6 +408,12 @@ Secondly, by default, `webpack-dev-server` will only auto-restart when it detect
 
 Once set up, `npx webpack serve` will host our web page on `http://localhost:8080/`, which we can open in our browser and start working!
 
+<div class="lesson-note">
+  
+Note that the webpack-dev-server reads your webpack configuration <em>once</em> when you start it. If you change the webpack config file while the dev server is running, it will not reflect those config changes. Use <kbd>Ctrl</kbd> + <kbd>C</kbd> to restart it to apply the new config.  
```

Comment:
Some wording suggestions. For future reference, if you want emphasis on a word we use markdown for that - surround with `*` for emphasis, surround with `**` for strong.

```suggestion
Note that the webpack-dev-server only reads your webpack configuration when you start it. If you change the webpack config file while the dev server is running, it will not reflect those config changes. Use <kbd>Ctrl</kbd> + <kbd>C</kbd> in the terminal to kill it then rerun `npx webpack serve` to apply the new config.  
```